devices.js

```diff
@@ -2240,6 +2240,31 @@ const devices = [
             execute(device, actions, callback);
         },
     },
+    {
+        zigbeeModel: ['motion'],
+        model: 'IM6001-MTP01',
+        vendor: 'SmartThings',
+        description: 'Motion sensor (2018 model)',
+        supports: 'occupancy and temperature',
+        fromZigbee: [
+            fz.generic_temperature, fz.ignore_temperature_change,
+            fz.ignore_iaszone_report, fz.ias_zone_motion_dev_change,
+            fz.bosch_ias_zone_motion_status_change, fz.generic_batteryvoltage_3000_2500,
```

Comment:
ias_zone_motion_dev_change and bosch_ias_zone_motion_status_change conflict with each other, I expect that one creates occupancy: false, while the other one corrects this with a occupancy: true. Can you check this?

Reply:
Yeah i can check that over this weekend.

Reply:
I tried with commenting out one and then the other.  The bosch version was the one providing the occupancy: true.  Addressed this in [new PR](https://github.com/Koenkk/zigbee-shepherd-converters/pull/373).